config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users
+  resources :posts
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
```

Comment:
- [ ] please make sure to not expose any unnecessary endpoints on your application for security, your routes should look something like below:
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

Reply:
thanks for pointing it out